db/migrate/20230126133557_convert_date_to_other_date.rb

```diff
@@ -0,0 +1,9 @@
+class ConvertDateToOtherDate < ActiveRecord::Migration[7.0]
+  def up
+    Page.where(answer_type: "date", answer_settings: nil).update_all(answer_settings: { input_type: "other_date" })
+  end
+
+  def down
+    Page.where(answer_type: "date", answer_settings: { input_type: "other_date" }).update_all(answer_settings: nil)
```

Comment:
I'm in 2 minds as to whether we should also perform the `down` migration on dates of birth - we'd be losing data  about what kind of date it is, but I'm not sure we ever want to be in a state where some dates are in the old format and some are in the new one:
```suggestion
    Page.where(answer_type: "date", answer_settings: { input_type: "other_date" }).update_all(answer_settings: nil)
    Page.where(answer_type: "date", answer_settings: { input_type: "date_of_birth" }).update_all(answer_settings: nil)
```

Reply:
Good point, I agree, we wouldn't want to be in a mix of formats (i.e find all date types and revert to "old format")

```suggestion
    Page.where(answer_type: "date").update_all(answer_settings: nil)
```